package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "url-polyfill",
```

Comment:
Not sure what the package name should be or if you intend to actually publish this. I'd be fine referencing it at its git url or whatever. `URL` is definitely taken already on pm.